charts/node-red/templates/deployment.yaml

```diff
@@ -52,6 +52,12 @@ spec:
             httpGet:
               path: {{ .Values.readinessProbePath }}
               port: http
+          startupProbe:
```

Comment:
Can we make this conditional?

startupProbe wasn't introduced until k8s 1.16 and will break those folks running prior to that version. 

Making it conditional will help with this possibility.